contracts/optimistic-ethereum/OVM/accounts/OVM_ECDSAContractAccount.sol

```diff
@@ -88,7 +95,7 @@ contract OVM_ECDSAContractAccount is iOVM_ECDSAContractAccount {
         // Contract creations are signalled by sending a transaction to the zero address.
         if (decodedTx.to == address(0)) {
             address created = Lib_SafeExecutionManagerWrapper.safeCREATE(
-                decodedTx.gasLimit - 2000,
+                decodedTx.gasLimit - MIN_GAS_LIMIT_PER_TRANSACTION,
```

Comment:
Looking at this again, I'm curious why `MIN_GAS_LIMIT_PER_TRANSACTION` is only deducted for a `CREATE` and not for a `CALL`?

What exactly is the purpose of  `MIN_GAS_LIMIT_PER_TRANSACTION`?

Reply:
It shouldn't be, you are correct. This issue will be taken care of in https://github.com/ethereum-optimism/contracts-v2/pull/85